src/Serilog/Formatting/Display/Padding.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System.IO;
+
+using Serilog.Parsing;
+
+namespace Serilog.Formatting.Display
+{
+    internal static class Padding
+    {
+        /// <summary>
+        /// Writes the provided value to the output, applying direction-based padding when <paramref name="alignment"/> is provided.
+        /// </summary>
+        public static void Apply(TextWriter output, string value, Alignment? alignment)
```

Comment:
Since the method deals with padding for anything and not just `Alignment`, passing an integer here and avoiding calls when there's no alignment specified seems clearer.